### DISCUSSION TITLE
Pagy Bootstrap Nav Pagination Menu Size

### DISCUSSION THREAD

**Discussion by pmichaeljones** [ddnexus/pagy#717] [Q&A] [answered]
Hi Pagy Fam,
I don't know if this is an issue, but it's an annoyance. 

When you're using the Bootstrap nav, the menu keeps jumping around in size.

For a good UX, the end user should be able to click "Next" continually, keeping the cursor in the same spot, without having to move the cursor every time to keep the cursor above the NEXT option.  You can see what I mean in the video below. 

Thoughts?

https://github.com/ddnexus/pagy/assets/6728265/4f04078f-c172-420f-a999-90910fa27658

**Comment by pmichaeljones** [comment, marked as answer]
Already found my answer: https://ddnexus.github.io/pagy/docs/how-to/#control-the-page-links

**Comment by ddnexus** [comment]
@pmichaeljones FYI: the latest version deprecates the legacy nav for the very reason you reported.

You can have a stable size, complete with first, last and gaps using the new default [fast nav](https://ddnexus.github.io/pagy/docs/how-to/#fast-nav).

Please, let us know if you like it.

**Reply by pmichaeljones** [reply]
Thanks @ddnexus. Can't say how impressed I am with Pagy and your insane docs. Best docs I've seen for a gem! 👍
